src/contracts/index.ts

```diff
@@ -1,8 +1,13 @@
 import { ethers } from "ethers";
 
-import { WolfSale, WolfSale__factory } from "./types";
+import {
+  AirWild2Sale,
+  AirWild2Sale__factory,
+  WolfSale,
+  WolfSale__factory,
+} from "./types";
 
-export * from "./types"
+export * from "./types";
 
 export const getWolfSaleContract = async (
```

Comment:
Is wolf sale still needed? Or can we just remove it?

Reply:
this was in a LOT of places, removed wolf sale entirely and that led to a lot more SDK changes